packages/system-upgrade-controller/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,23 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,9 +1,14 @@
+ apiVersion: v1
+-name: rancher-k3s-upgrader
+-description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+-  Users do not need to manually upgrade this app. It will be automatically upgraded to the latest version when upgrading a cluster.
+-home: https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader
++name: system-upgrade-controller
++description: General purpose controller to make system level updates to nodes
++home: https://github.com/rancher/system-charts/charts/system-upgrade-controller
+ sources:
+-  - "https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader"
+-version: 0.3.0
++  - "https://github.com/rancher/system-charts/charts/system-upgrade-controller"
++version: 0.7.4
```

Comment:
This patch will not make a difference since it is overridden by the `version` in the package.yaml (see produced Chart.yaml under `charts/` directory above). We might as well remove it.